PWGHF/D2H/Macros/compute_fraction_cutvar.py

```diff
@@ -155,30 +172,32 @@ def main(config):
             hist_frac_raw_nonprompt.SetBinContent(ipt + 1, raw_frac_nonprompt[0])
             hist_frac_raw_nonprompt.SetBinError(ipt + 1, raw_frac_nonprompt[1])
 
-        canv_rawy, histos_rawy, leg_r = minimiser.plot_result(f"_pt{pt_min:.0f}_{pt_max:.0f}")
+        hist_bin_title = "bin # " + str(ipt+1) + f"; {pt_axis_title}#in (" + str(pt_min) + "; " + str(pt_max) + ")"
```

Comment:
I would suggest to use a single f-string (easier to read :-))
```suggestion
        hist_bin_title = f"bin # {ipt+1}; {pt_axis_title}#in ({pt_min:.0f}; {pt_max:.0f})"
```

Reply:
Done